contracts/BackedTokenImplementation.sol

```diff
@@ -72,6 +73,9 @@ contract BackedTokenImplementation is OwnableUpgradeable, ERC20PermitDelegateTra
     // SanctionsList:
     SanctionsList public sanctionsList;
 
+    // Terms:
+    string public terms;

```

Comment:
You should update the __gap at the end of this file to be 49 instead of 50 (although if no contract is inheriting from this one the __gap isn't even necessary).

Reply:
Hi @ben-kaufman. You are right. @alexander2001i also mentioned it. 
At the moment no contract is inheriting it. So it does not really matter, it is there for future use. But I will update it.

10x.